.github/workflows/release.yml

```diff
@@ -21,12 +21,35 @@ jobs:
         with:
           fetch-depth: '0'
 
-      - name: Merge develop
+      - name: branch change and package version update
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
+          git config --local user.email "github-actions@github.com"
+          git config --local user.name "GitHub Actions"
+          git checkout develop
+          npm version --no-git-tag-version --no-commit-hooks ${{ inputs.next_version }}
+
+      - name: Modify algoliaconfig file
+        run: |
+          sed -i 's/"version": ".*"/"version": "${{ inputs.next_version }}"/' cartridges/int_algolia/algoliaconfig.json
+
+      - name: Commit changes
+        run: |
+          git add .
+          git commit -m "Update version to ${{ inputs.next_version }}"
+          git push
+
+      - name: Merge develop into master
+        run: |
+          set -e
+          echo "Checking out develop branch..."
+          git checkout develop
+          git pull --ff-only
```

Comment:
`--ff-only` is the default, did you explicitly need it?
(And you've already checked out develop before, it shouldn't be needed)

Reply:
Yes, you're right. When I initially wrote this code piece, I didn't checkout to develop, and that part remained from the initial version.

Removing it.